### PR TITLE
Anonymous function naming

### PR DESCRIPTION
**Pull request by jprestwo**

### CODE REVIEW

File: src/zjs_aio.c [hunk 1]
```diff
@@ -233,6 +233,10 @@ static ZJS_DECL_FUNC(zjs_aio_pin_on)
             return zjs_error("could not allocate handle");
 
         jerry_set_object_native_pointer(this, handle, &aio_type_info);
+#ifdef ZJS_FIND_FUNC_NAME
+    zjs_obj_add_string(argv[0], "aio: onchange",
```
Comment (grgustaf) [inline src/zjs_aio.c]: argv[0] is a string event name... you want argv[1] *if* it's a function

File: src/zjs_callbacks.c [hunk 2]
```diff
@@ -576,7 +576,7 @@ void zjs_call_callback(zjs_callback_id id, const void *data, uint32_t sz)
 #ifdef DEBUG_BUILD
                         DBG_PRINT("callback %d had error; creator: %s, caller: %s\n", id, cb_map[id]->creator, cb_map[id]->caller);
 #endif
-                        zjs_print_error_message(rval, cb_map[id]->js_func);
+                        zjs_print_error_message(rval, cb_map[id]->func_list[i]);
```
Comment (grgustaf) [inline src/zjs_callbacks.c]: this was a latent bug?

File: src/zjs_util.c [hunk 3]
```diff
@@ -406,9 +406,14 @@ static char *create_js_path(char *obj_name, name_element_t *parent)
...
     int total = strlen(obj_name) + 1;
+
+    if (!obj_name) {
+        return NULL;
+    }
+
     char *str = zjs_malloc(total);
     if (!str) {
-        return "";
+        return NULL;
```
Comment (grgustaf) [inline src/zjs_util.c]: good catch, i could have sworn i changed that already

File: src/zjs_callbacks.c (hunk 2 above)
Comment (jprestwo) [inline src/zjs_callbacks.c]: Yes, yes it was :)

**Comment by grgustaf** [comment]
+1

**Pull request merged**